yktrace/src/maptrace.rs

```diff
@@ -0,0 +1,143 @@
+use object::Object;
```

Comment:
This whole file is hardware tracer specific, so I wonder if we should have:
 - src/
   - swt.rs
   - hwt/
     - mod.rs (was hwt.rs)
     - mapper.rs